src/functions-and-arrays.js

```diff
@@ -52,16 +148,71 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+let uniqueArray = [];
 
+function uniquifyArray(wordsUnique) {
+  for (let z = 0; z < wordsUnique.length; z++) {
+    if (uniqueArray.indexOf(wordsUnique[z]) === -1) {
+      // if indexOf can't find the word, it returns -1
+      uniqueArray.push(wordsUnique[z]);
+    }
+  }
+  return uniqueArray;
+}
 
+uniquifyArray(wordsUnique);
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+let keyword = 'truth';
+let t = 0;
+let itsTrue = 0;
+let itsFalse = 0;
+
+function doesWordExist(wordsFind, keyword) {
+  while (itsTrue === 0) {
+    if (wordsFind[t].includes(keyword)) {
```

Comment:
Here, you need to check if keyword is inside wordsFind array, so you can write :
if(wordsFind[t] == keyword){
 return true;
  }
}

Reply:
Hey, thanks for the feedback, I fixed all the errors for the mandatory iterations now!